src/telegramWebHook.js

```diff
@@ -133,8 +135,13 @@ class TelegramBotWebHook {
     debug('WebHook request URL: %s', req.url);
     debug('WebHook request headers: %j', req.headers);
 
-    // If there isn't token on URL
-    if (!this._regex.test(req.url)) {
+    if (this._healthRegex.test(req.url)) {
+      // If this is a health check
+      debug('WebHook health check passed');
+      res.statusCode = 200;
+      res.end('OK');
+    } else if (!this._regex.test(req.url)) {
```

Comment:
I think we should test for the token **first**, then the health endpoint, considering that we do **not** want to waste too much CPU with each and every update we receive on the webhook! What do you think?